### DISCUSSION TITLE
No stack trace in paddle

### DISCUSSION THREAD

**Discussion by tzktz** [PaddlePaddle/PaddleOCR#12294] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Ubuntu
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：paddleocr latest
- 运行指令/Command Code：
- 完整报错/Complete Error Message：--------------------------------------
C++ Traceback (most recent call last):
--------------------------------------
No stack trace in paddle, may be caused by external reasons.
----------------------
Error Message Summary:
----------------------
FatalError: `Termination signal` is detected by the operating system.
[TimeInfo: *** Aborted at 1711954022 (unix time) try "date -d @1711954022" if you are using GNU date ***]
[SignalInfo: *** SIGTERM (@0x7) received by PID 62 (TID 0x7f9fcf01ab80) from PID 7 ***]

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by cuicheng01** [comment]
This error is difficult to locate. You can reinstall paddlepaddle and try again.

**Comment by tzktz** [comment]
> This error is difficult to locate. You can reinstall paddlepaddle and try again.

i have tried a lot...bt no effect same error.. @cuicheng01

**Comment by cuicheng01** [comment]
Try changing to the official docker image?

**Comment by tzktz** [comment]
> Try changing to the official docker image?

build:
  gpu: false
  python_version: "3.8"
  system_packages:
    - "ffmpeg"
    - "libsm6"
    - "libxext6"
    - "libgl1-mesa-glx"
    - "poppler-utils"
  python_packages:
    - torch
    - torchvision
    - transformers
    - pillow
    - numpy
    - pandas
    - opencv-python
    - PyMuPDF
    - xlsxwriter
    - beautifulsoup4
    - paddlepaddle
    - paddleocr==2.7.0.3

predict: "predict.py:Predictor"    
this is my cog.yaml file for docker image @cuicheng01

**Comment by UserWangZz** [comment]
I see GPU false, is it using the CPU version
